lib/README.md

```diff
@@ -50,12 +50,12 @@ After that, add and commit the additional files that were created within lib/
 - from https://github.com/rhwood/jsplitbutton
 - javadoc at https://www.javadoc.io/doc/com.alexandriasoftware.swing/jsplitbutton/1.3.1
 
-##### jinputvalidator-0.3.0.jar
-- version 0.3.0
+##### jinputvalidator-0.4.0.jar
+- version 0.4.0
 - provides validation for JComponents
 - contributed by Randall Wood
 - from https://github.com/rhwood/jinputvalidator
-- javadoc at https://www.javadoc.io/doc/com.alexandriasoftware.swing/jinputvalidator/0.3.0
+- javadoc at https://www.javadoc.io/doc/com.alexandriasoftware.swing/jinputvalidator/0.4.0
```

Comment:
A simple note:
When opening the new link, I get the error:
Version 0.4.0 does not exist for artifact com.alexandriasoftware.swing:jinputvalidator. >> Visit latest version 0.3.0

Reply:
javadoc.io takes up to 24 hours to pull new versions of Javadoc JARs